src/backend/distributed/planner/relation_restriction_equivalence.c

```diff
@@ -155,6 +163,11 @@ static bool RangeTableArrayContainsAnyRTEIdentities(RangeTblEntry **rangeTableEn
 													queryRteIdentities);
 static Relids QueryRteIdentities(Query *queryTree);
 
+static bool FindQueryContainingRTEIdentityInternal(Node *node,
```

Comment:
minor: we typically order the functions the other way, first the one that uses the other one(s)